adding_species_manuscript.tex

```diff
@@ -453,34 +421,65 @@ \subsection*{Additions to \stdpopsim}
+In bacteria and archaea, homologous recombination occurs primarily through gene conversion rather than single crossover recombinations.
+As a result, such species cannot be realistically simulated with a recombination model that considers only crossovers.
+Gene conversion in such species is implemented in \stdpopsim by setting the
+\texttt{bacterial\_recombination} flag to \texttt{True} (as a feature of the genome)
+and the  \texttt{gene\_conversion\_length} parameter to the average gene conversion tract length.
+This will result in all recombinations being simulated via gene conversion
+(no crossover recombinations), where the tract length is sampled from a geometric
+distribution whose mean is the specified length.
+For example, the model for \textit{Escherichia coli} has been updated in the \stdpopsim catalog to have a gene conversion rate of $8.9\times 10^{-11}$ with an average tract length of 345 bases \citep{Wielgoss2011,Didelot2012}.
+%
+Some species undergo recombination by gene conversion as well as crossover recombination.
+To accommodate this in \stdpopsim simulations,
+one needs to set two additional parameters in each chromosome: \texttt{gene\_conversion\_fraction},
+which specifies the fraction of recombinations that occur due to gene conversion;
+and \texttt{gene\_conversion\_length}, which is the average tract length, as defined above. For example, the model for \emph{Drosophila melanogaster} has been updated in the \stdpopsim catalog to have a fraction of gene conversions of 0.83 (in all chromosomes with recombination) and an average tract length of 518 bases \citep{Comeron2012}.
+% ILAN; I think that the part below is no longer needed. Please check.
+%
+% We note, however, that in eukaryotes, the gene conversion rate might be considerably higher than the rate of single crossover recombination.
+%As a result, simulations of eukaryote genomes with gene conversion will typically incur a significant computational load.
+%Therefore, in \stdpopsim, users are required to explicitly enable gene conversion
+%by  using the use\_species\_gene\_conversion flag.
 % Elise: Not sure the blow fits in the GC paragraph?
-In addition, the update to msprime version 1.0~\citep{Baumdicker2022}
-provides a number of additional benefits such as a discrete site model of mutation,
-so that simulated data will now, as in real data,
-have a small proportion of sites with multiple mutations and more than two alleles.
-
-Moreover, we have extended \stdpopsim so that
-genome annotations can be associated with a genome assembly.
-These can be used to simulate selection at a subset of sites (e.g., the annotated coding regions)
+% Ilan: the idea is that the model of GC is enabled due to the upgrade to msprime 1.0. I made sure to mention this above. This is still a bit tacked-on here, but still probably the best place to mention this.
```

Comment:
I think we should mention here that GC is *not* on by default because it makes things significantly slower/bigger,.

Reply:
Done